scripts/generate-clients/copy-to-clients.js

```diff
@@ -8,30 +8,14 @@ const getOverwritableDirectories = (subDirectories, packageName) => {
     "@aws-sdk/client-sts": ["defaultRoleAssumers.ts", "defaultStsRoleAssumers.ts", "defaultRoleAssumers.spec.ts"],
   };
   const overwritableDirectories = [
-    "commands",
-    "models",
-    "protocols",
-    "pagination",
-    "tests",
-    "waiters",
+    "src", // contains all source files
     "LICENCE",
-    "runtimeConfig.ts",
```

Comment:
'jest.config.js' can be removed in following PR, if the client doesn't include unit test

Reply:
The command to run tests using jest was added when protocol_tests were written in https://github.com/awslabs/smithy-typescript/commit/6b6a3b182771e00742369e949903087378862ea2

I'll keep `jest.config.js` as of now, till we revisit whether we should recommend testing framework in generated client code.

We can remove `jest.config.js` and `jest-*` dependencies from clients in aws-sdk-js-v3, but we use jest for other testing so it's presence doesn't affect the repository.